src/modules/admin/components/users/UserAuditHistory.tsx

```diff
@@ -0,0 +1,51 @@
+import { Alert } from '@mui/material';
+import { ContextualCollapsibleListsProvider } from '@/components/elements/CollapsibleListContext';
+import useSafeParams from '@/hooks/useSafeParams';
+import {
+  AuditHistoryNode,
+  auditHistoryColumns,
+} from '@/modules/audit/components/auditHistoryColumnDefs';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import {
+  UserAuditEventFilterOptions,
+  GetUserAuditEventsDocument,
+  GetUserAuditEventsQuery,
+  GetUserAuditEventsQueryVariables,
+} from '@/types/gqlTypes';
+
+const UserAuditHistory = () => {
+  const { userId } = useSafeParams() as { userId: string };
+  // On the user audit history page, we don't need to show the user column, since the value will always be the same
+  const columns = auditHistoryColumns.filter((column) => column.key !== 'user');
+
+  return (
+    <>
+      <Alert severity='warning' sx={{ m: 3 }}>
+        Data on this page may be delayed by up to an hour. Recent user activity
+        may not be immediately visible.
+      </Alert>
```

Comment:
This alert is actually supposed to go on the other card ("user access"). User edit history should be available immediately.

Reply:
Sorry I missed this - thanks!